### PR TITLE
physics-simulation done

### PR DESCRIPTION
**Pull request by Mk996**
PR Details:
- This PR has new changes for the physics-simulation section on the home page.

### CODE REVIEW

**Comment by Mk996** [comment]
@tvcastillod @ganimtron-10 Please review this

**Comment by ganimtron-10** [comment]
Hey @Mk996 would suggest you to directly add us as reviewers from the left hand side section called Reviews instead of commenting for the review!

**Comment by Mk996** [comment]
> Hey @Mk996 would suggest you to directly add us as reviewers from the left hand side section called Reviews instead of commenting for the review!

Hey @ganimtron-10, so I didn't have access to it. I just got it now will take care from the next PRs

File: docs/source/_templates/home.html
Comment (skoudoro) [inline docs/source/_templates/home.html:441]: This text is slightly too big in mobile version. Can you update this text during mobile version.

Also, it would be great to use the same font family as the other section (scientific domains, etc...)

**Review by skoudoro — COMMENTED** [review]
LGTM, thanks @Mk996.

See below my small comment, after that, it can be merged

**Comment by JoaoDell** [comment]
Hey @Mk996, I don't have much (almost none) practice with html and css, so I don't think I will be much effective reviewing your PRs, but will see what I can do 👍

**Comment by ganimtron-10** [comment]
Hey @Mk996 ,
Did you tried building it locally I am getting some errors while building it!

**Comment by Mk996** [comment]
> ###
> [Jul 7, 2023](#issuecomment-1625666847)

@ganimtron-10 Every new change is reflected upon re-building only. I have tested that on my system @skoudoro can you try once

**Comment by ganimtron-10** [comment]
Then I think there is some issue with my system as I am getting `URLError` with some more client side error while fetching things.

File: docs/source/_templates/home.html
Comment (ganimtron-10) [inline docs/source/_templates/home.html:529]: Why so few links are using `raw.githubusercontent` and few directly linking to the `fury-communication-assets`?
Can't we use consistent links?

Also please have a look at Github permalinks it would help here!

File: docs/source/_templates/home.html
Comment (ganimtron-10) [inline docs/source/_templates/home.html:551]: same as above

**Review by ganimtron-10 — CHANGES_REQUESTED** [review]
Hey @Mk996,
Thanks for this PR. I can confirm that the website building issue is totally on my local machine, not sure why I am unable to load `rawgithubusercontent` but the workaround we discussed in yesterday's meeting helped me in that.

Please have a look at the below comment, once done this PR is good to go!

File: docs/source/_templates/home.html
Comment (Mk996) [reply docs/source/_templates/home.html:529]: I did not change any link from the existing code. If not I might need to see what to change.

Comment (Mk996) [reply docs/source/_templates/home.html:529]: Also, using the raw GitHub link would be a better option as the fury-communications-asset is meant for storing the assets.

We don't want to go around and create a permalink of the objects as they are hard to manage.

File: docs/source/_templates/home.html
Comment (Mk996) [reply docs/source/_templates/home.html:551]: This cannot be converted to raw user link or permalink as the size of the asset is too big

**Comment by Mk996** [comment]
We might need to hold this PR till https://github.com/fury-gl/fury/pull/820 gets merged as the new interface for the mobile version of the scientific domain also redirects to tutorial corresponding to the video played.

File: docs/source/_templates/home.html
Comment (ganimtron-10) [reply docs/source/_templates/home.html]: Yup, I also think raw GitHub links might help.
Let's see what @skoudoro has to say!

**Comment by ganimtron-10** [comment]
Hey @skoudoro, This PR is almost ready please check this out!

File: docs/source/_templates/home.html
Comment (skoudoro) [reply docs/source/_templates/home.html]: no need for permalink, we keep permalink only for code for now.

Otherwise, I am ok, for both version, they point to the same place.

**Review by skoudoro — COMMENTED** [review]
All good, you just need to update some link @Mk996.  https://github.com/fury-gl/fury/pull/820 has been merged so I am just waiting for you to update those links and we can merge this PR.

Thanks !

**Comment by skoudoro** [comment]
ping @Mk996, Can you update some links, then we can merge this PR

**Review by skoudoro — APPROVED** [review]
Thank you for the update. LGTM. merging

**Pull request merged**
